wgengine/wgengine.go

```diff
@@ -59,7 +60,7 @@ type Engine interface {
 	// sends an updated network map.
 	//
 	// The returned error is ErrNoChanges if no changes were made.
-	Reconfig(cfg *wgcfg.Config, dnsDomains []string, localSubnets []wgcfg.CIDR, noSNAT bool) error
+	Reconfig(cfg *wgcfg.Config, routerCfg *router.Config) error
```

Comment:
I'd just drop the parameter names now (cfg and routerCfg). Also, the docs mention "The provided DNS domains" which is no longer applicable.